### PR TITLE
PETALSSEFLOWABLE-47

### PR DESCRIPTION
**Pull request by cdeneux**
ProcessInstanceService Searching for a process instance with a variable type different from a string does not work

### CODE REVIEW

File: src/test/java/org/ow2/petals/flowable/ServiceProviderVacationProcessTest.java [hunk 1]
```diff
@@ -115,14 +114,12 @@ public static void assertTechnicalMonitoringMetrics()
...
-         * 5 process instances in state 'Pending', the ones created by unit tests:
-         * - userTaskRequest_EmptyProcessInstanceIdValue,
-         * - userTaskRequest_EmptyUserIdValue,
-         * - userTaskRequest_NoUserIdValue,
-         * - userTaskRequest_NoProcessInstanceIdValue,
-         * - userTaskRequest_TaskCompletedFault
+         * 5 process instances in state 'Pending', the ones created by unit tests: -
+         * userTaskRequest_EmptyProcessInstanceIdValue, - userTaskRequest_EmptyUserIdValue, -
+         * userTaskRequest_NoUserIdValue, - userTaskRequest_NoProcessInstanceIdValue, -
+         * userTaskRequest_TaskCompletedFault
```
Comment (alagane) [inline src/test/java/org/ow2/petals/flowable/ServiceProviderVacationProcessTest.java]: Formatting is broken, you should keep new lines here.

File: src/test/java/org/ow2/petals/flowable/incoming/integration/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) 2018 Linagora
```
Comment (alagane) [inline src/test/java/org/ow2/petals/flowable/incoming/integration/UtilsTest.java]: I think the year should be 2019.

Comment (cdeneux) [reply src/test/java/org/ow2/petals/flowable/incoming/integration/UtilsTest.java]: Sure, but we are on a branch whith coding rules based on 2018. Copyright date is moved to 2019 on branch 1.1.x

File: src/test/java/org/ow2/petals/flowable/ServiceProviderVacationProcessTest.java (hunk 1 above)
Comment (cdeneux) [reply src/test/java/org/ow2/petals/flowable/ServiceProviderVacationProcessTest.java]: Was fixed merging changes

**Pull request merged**